kit/ChildSession.cpp

```diff
@@ -1113,10 +1113,16 @@ bool ChildSession::downloadAs(const char* /*buffer*/, int /*length*/, const Stri
             (format.empty() ? "(nullptr)" : format.c_str()) << "', ' filterOptions=" <<
             (filterOptions.empty() ? "(nullptr)" : filterOptions.c_str()) << "'.");
 
-    getLOKitDocument()->saveAs(url.c_str(),
+    bool success = getLOKitDocument()->saveAs(url.c_str(),
                                format.empty() ? nullptr : format.c_str(),
                                filterOptions.empty() ? nullptr : filterOptions.c_str());
 
+    if (!success)
+    {
+        LOG_ERR("SaveAs Failed for id=" << id << " [" << url << "].");
+        return false;
```

Comment:
I think we should send some error message here, client shows 'busy popup' on save as, we should hide that and show error for a user

Reply:
maybe this getError in the error handling code will provide more info: https://github.com/CollaboraOnline/online/blob/f88a38a4e7d25f1d34350b313df42f3ae0acbe76/kit/ChildSession.hpp#L345

Reply:
`getLOKitDocument()->getError()` should work I think?

Reply:
thanks